locale/it-IT.json

```diff
@@ -0,0 +1,167 @@
+{
+  "common": {
+    "countdown": "{days, plural, =0 {Solo {time} rimanente} =1 {# giorno {time}} other {# giorni {time}}}",
+    "load_more": "Carica altro",
+    "cancel": "Cancella",
+    "submit": "Sottoscrivi",
+    "undo": "annulla"
+  },
+  "titles": {
+    "categories": "Categorie",
+    "filter": "Filtro",
+    "cart": "Carrello",
+    "shipping": "Spedizione",
+    "reviews": "Recensioni",
+    "browse": "Sfoglia",
+    "favorites": "Favoriti"
+  },
+  "favorites": {
+    "add": "Aggiungi ai favoriti",
+    "continue": "Continua ad acquistare",
+    "empty": "Non hai favoriti",
+    "error_add": "Non Ë stato possibile aggiungere il prodotto ai favoriti",
+    "error_remove": "Non Ë stato possibile rimuovere il prodotto dai favoriti",
+    "remove": "Rimuovi dai favoriti",
+    "removed": "Rimosso dai favoriti",
+    "modal": {
+      "message": "Per aggiungere questo prodotto al tuo carrello, prego selezionare le varianti.",
+      "confirm": "Seleziona le varianti"
+    }
+  },
+  "filter": {
+    "sort": {
+      "default": "Raggruppa per",
+      "most_popular": "pi˘ popolare",
+      "price_desc": "Prezzo: Dall'alto in Basso",
+      "price_asc": "Prezzo: Dal Basso in Alto"
+    },
+    "apply": "Applica",
+    "more": "altro",
+    "clear": "Resetta",
+    "clear_all": "Resetta tutto",
+    "view_all": "Vedi tutto"
+  },
+  "price": {
+    "from": "Da {price}",
+    "range": "Prezzo da {fromPrice} a {toPrice}",
+    "tier": {
+      "set": "{from} elementi: {price}",
+      "range": "{from} o altro: {price}"
+    },
+    "msrp": "SRP",
+    "currency": "Dollari"
+  },
+  "product": {
+    "description_heading": "A riguardo",
+    "pick_an_attribute": "Seleziona un {0}",
+    "tax_disclaimer": "* Incl. IVA pi˘ costi di spedizione"
```

Comment:
here is a comma missing. please use a json validator to make sure the file is correct